kairon/shared/actions/utils.py

```diff
@@ -501,6 +501,21 @@ def prepare_response(response_template: str, http_response: Any):
 
         return parsed_output
 
+    @staticmethod
```

Comment:
context contains the current state for a user(run time slot values, user message, sender id ,etc). we have to pass this context along with source code so that the user can make use of it. Slots can be modified too.
Another review comment given in the last PR was to prevent setting kairon system slots.